buildpacks/jvm-function-invoker/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "jvm-function-invoker-buildpack"
+version = "0.1.0"
+authors = ["Terence Lee <hone02@gmail.com>"]
```

Comment:
```suggestion
```

The authors field is optional. Seeing that (a) we're not publishing the buildpack to crates.io (since it's not a library), (b) we're not going to keep this up to date, (c) it's not ideal to leak personal email addresses, I think we should just omit the field.